website/docs/components/carousel/carousel-a11y.md

```diff
@@ -22,25 +22,30 @@ The list below describes roles and attributes that component already has.
-|                                     | `aria-current="active"`                       | Indicates that this element represents the current item within a container or set of related elements.                                                                               |
-| `Carousel.Prev`                     | `aria-controls="carouselID"`                  | Refers to the tabpanel element associated with the tab.                                                                                                                              |
-|                                     | `aria-label="Previous slide"`                 | Defines a label for the button that opens previous list item.                                                                                                                        |
-| `Carousel.Next`                     | `aria-controls="carouselID"`                  | Refers to the tabpanel element associated with the tab.                                                                                                                              |
-|                                     | `aria-label="Next slide"`                     | Defines a label for the button that opens next list item.                                                                                                                            |  |
-| `Modal.Close`                       | `aria-label="Close"`                          | Provides an accessible name for the **Close** button inside the Modal window.                                                                                                        |
+| Component / element   | Role & attributes                             | Usage                                                                                                                                                                                |
+| --------------------- | --------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `Carousel`            | `role="region"`, `roledescription="carousel"` | Defines the carousel and its controls as a landmark region. `roledescription` informs assistive technologies to identify the element as a "carousel".                                |
+| `Carousel.Indicators` | `role="tablist"`                              | Indicates that the element serves as a container for a set of tabs.                                                                                                                  |
+|                       | `aria-label="Slides"`                         | Adds a label to a container for a set of tabs.                                                                                                                                       |
```

Comment:
@j-mnizhek I've been thinking about this recently, wouldn't the empty cells be an a11y issue because it's not clear that we mean a merged cell here?

Reply:
This is an interesting question! Let me consult with Glasha first, maybe she have had something similar in her experience